docs/sources/reference/components/prometheus/prometheus.remote_write.md

```diff
@@ -404,6 +404,37 @@ prometheus.remote_write "default" {
 }
 ```
 
+## Troubleshooting
+
+### Out of order errors
+
+You may sometimes see an "out of order" error in {{< param "PRODUCT_NAME" >}}'s logs.
```

Comment:
```suggestion
You may sometimes see an "out of order" error in the {{< param "PRODUCT_NAME" >}} log files.
```

We should try to avoid making Alloy possessive (using `Alloy's`)

Reply:
I keep making this mistake :) Thanks for correcting me again!